app/controllers/application_controller.rb

```diff
@@ -2,4 +2,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+private
+
+  def error_unprocessable_entity
+    render \
+      file: Rails.root.join('public', '422.html'),
+      layout: false,
+      status: 422
```

Comment:
This whole method feels a bit… odd… to me, and I can't quite place why. Perhaps it should be handled by raising an exception? I suspect my sense is a bit off here, though. :confused:

Separately, it could be good to replace the `422` with `:unprocessable_entity`? Rails will handle that equivalently to the status code, and it seems more descriptive that way?